SubscribeIT/ngDesk
# 147-Escalations-get-calls-to-be-replaced-with-graphql-calls-in-the-UI

Closes #158 

Test Case 1

Name: Check the graphql calls for escalation list layout

1. Login to ngdesk.
2. Goto -> Pager -> Escalations -> New.
3. Create a new escalation and hit save.
4. Observe that in the network request the created escalation is fetched from the graphql calls instead of ngdesk-rest call.
5. Observe no error.

Test Case 2

Name: Check the graphql calls for escalation edit layout

1. Login to ngdesk.
2. Goto -> Pager -> Escalations.
3. Open the created escalation.
4. Observe that in the network request the created escalation detail is fetched from the graphql calls instead of ngdesk-rest call.
5. Observe no error.

![Screenshot from 2021-09-07 09-39-28](https://user-images.githubusercontent.com/89504233/132282994-6d20d330-45df-48fc-ae8b-872ba2eb37bf.png)
![Screenshot from 2021-09-07 09-39-52](https://user-images.githubusercontent.com/89504233/132283000-e7487c73-ee8d-4273-b4f3-dff9b8236721.png)
